include/ozo/impl/read.h

```diff
@@ -23,10 +23,12 @@ inline Require<RawDataWritable<T>, istream&> read(istream& in, T& out) {
 
 template <typename T>
 inline Require<Integral<T> && sizeof(T) == 1, istream&> read(istream& in, T& out) {
-    out = in.get();
+    istream::traits_type::int_type c = in.get();
+    // TODO: Check result of istream::traits_type::not_eof(c)
```

Comment:
Thanks, but could you please remove this comment and open an appropriate issue instead. I do not believe in TODO comments unfortunately they have no chances to be performed in real life in opposite to ticket or issues, which are always on the board.

Thank you!

Reply:
I'm happy to work with you on this, but I think we're getting a little too deep into the minutia here.

You could have created the github issue that you want me to create, and merged this change as is (with or without the TODO: comment)

Instead, we've added another round trip between your team, and mine.

Note that I consider #187 a minor problem. I'm much more concerned with #181 . I'm still unable to use OZO to execute a query in my code. There are so many parts of OZO that are not easy to understand as an end user.